package.json

```diff
@@ -10,7 +10,7 @@
     "build:dist": "NODE_ENV=production webpack --config webpack.config.production.js",
     "lint": "eslint --ext .js,.jsx src",
     "deploy": "npm run build:dist && surge -p .",
-    "start": "node server.js"
+    "start": "NODE_ENV=development node server.js"
```

Comment:
@kenwheeler -- I think just add `builder` alone to get the extra's like `builder envs` and `builder concurrent` and not worry about an archetype since spectacle itself is kind of a "one-off".

The `spectacle-boilerplate` could definitely be replaced by an archetype like `builder-spectacle` now that `builder-init` has landed.

That's my gut intuition, but it _may_ be the case we could add `init/` templates and `init.js` to `spectacle` itself and just have it all together.